### PR TITLE
EVSS: Make sure updated_at is set correctly

### PR DESCRIPTION
**Pull request by carlmjohnson**
Fixes https://github.com/department-of-veterans-affairs/sunsets-team/issues/117

### CODE REVIEW

**Comment by carlmjohnson** [comment]
Once we merge this, we can just copy-pasta https://github.com/department-of-veterans-affairs/vets-api/pull/364/files to make a reaping task that looks at updated_at.

File: app/services/disability_claim_service.rb [hunk 1]
```diff
@@ -90,7 +90,7 @@ def claims_scope
 
   def create_or_update_claim(raw_claim)
     claim = claims_scope.where(evss_id: raw_claim['id']).first_or_initialize(data: {})
-    claim.update_attributes(data: claim.data.merge(raw_claim), successful_sync: true)
```
Comment (carlmjohnson) [inline app/services/disability_claim_service.rb]: I am starting to wonder if this `data.merge` might have caused some of the issues that Josh saw with data not clearing.

Comment (austin-martinez) [inline app/services/disability_claim_service.rb]: I think I had this backwards in our code now, but the way I was thinking about it was that the details call would always replace the data we have for the claim as it has the most data. The list call should only update the fields it contains if the user goes back and new data is fetched. Looking at it now, there are some nested hashes that are probably causing issues when going back to the list view and wiping out the attribute from the details. The end goal is that after making the list and details calls, we should always be able to reconstruct both views from the data in the DB. In your first scenario, I think that should cause an error, if the claim only contains data from the EVSS list call, there is not enough info for the details page. Quick solution may be storing 2 JSON blobs and throwing an error on the details page when EVSS is down and we don't have the details blob.

**Comment by austin-martinez** [comment]
There is definitely an issue with the merging of data from the list and details, but I think the issue this PR is addressing is no longer relevant. The date should be removed from the banner rather than changing how `updated_at` works. If we foresee needing to know the date of the last sync within the 24 hours the data is persisted for, we should add another timestamp column and not overload the `updated_at` timestamp

File: app/models/disability_claim.rb [hunk 2]
```diff
@@ -5,4 +5,9 @@ class DisabilityClaim < ActiveRecord::Base
   attr_accessor :successful_sync
 
   scope :for_user, ->(user) { where(user_uuid: user.uuid) }
+
+  def update_evss_data(raw_claim)
+    update_attributes(data: raw_claim, successful_sync: true)
+    touch # Ensure that updated_at is touched
```
Comment (carlmjohnson) [inline app/models/disability_claim.rb]: >  I don't think we want to change updated_at if the underlying data hasn't changed 

My understanding of https://github.com/department-of-veterans-affairs/sunsets-team/issues/117 was that we wanted to force an updated_at. If we want a separate column for last_access to use for TTL, we could change this to `touch :last_access`. I don't know if there's a way to consolidate DB calls though.

**Pull request closed without merging**